definitions/divorce/json/CaseEventToFields.json

```diff
@@ -9109,5 +9109,18 @@
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "23/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solApplyForDA",
+    "CaseFieldID": "ApplyForDecreeAbsolute",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "petsolapplyforda",
+    "PageLabel": "Do you want to finalise your divorce?",
```

Comment:
Content issue, this probably is not the solicitor divorce.

Reply:
Can this change to "**Do you want to apply to finalise the divorce?**"